media/css/firefox/whatsnew/whatsnew-95-de.scss

```diff
@@ -0,0 +1,183 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-pocket';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/components/picto';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import 'includes/header';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.wnp-content-main {
+    padding: 0;
+    text-align: center;
+}
+
+.wnp-main-image {
+    display: block;
+    margin: 0 auto $layout-sm;
+}
+
+.wnp-main-title {
+    @include text-title-md;
+    color: get-theme('title-text-color');
+}
+
+.wnp-main-tagline {
+    @include text-body-lg;
+}
+
+.wnp-main-cta {
+    margin-top: $layout-md;
+}
+
+.mzp-c-logo {
+    margin: 0 auto $spacing-xl;
+}
+
+// * -------------------------------------------------------------------------- */
+// Extra content
+
+.mzp-c-picto {
+    text-align: center;
+    max-width: $content-sm;
+
+    @media #{$mq-lg} {
+        @include bidi(((text-align, left, right),));
+    }
+}
+
+.mzp-c-picto-image svg {
+    height: auto;
+    max-width: 100%;
+}
+
+.wnp-content-footer {
```

Comment:
I don't think this is used in the DOM either?

Reply:
Will remove 👍